File: HackerEarth/Circuit/2.js
process.stdin.resume();
process.stdin.setEncoding("utf-8");
var stdin_input = "";

process.stdin.on("data", function(input) {
  stdin_input += input;
});

process.stdin.on("end", function() {
  main(stdin_input);
});

const factorial = n => {
  let mu = 1000000007;
  let fact = 1;
  while (n > 0) {
    fact *= n;
    fact = fact % mu;
    n--;
  }
  return fact % mu;
};

function main(input) {
  const inputString = stdin_input.split(" ");
  let [n, m] = inputString;
  n = Number(n);
  m = Number(m);

  const result = factorial(m);

  process.stdout.write(result + "\n");
}

File: HackerEarth/Circuit/2.py
import math
n = input()
n = n.split(" ")
x = []
for j in n:
    x.append(int(j))

b1 = int(x[1]*x[2])
b2 = int(x[0]*x[3])
b3 = int((b2*x[1])/x[3])

g = math.gcd(b1,math.gcd(b2,b3))
b1=int(b1/g)
b2=int(b2/g)
b3=int(b3/g)

print("{} {} {}".format(b1,b2,b3))